### PR TITLE
This is the new Risk Score

### PR DESCRIPTION
**Pull request by therisingpage**
@skellerfenix Hello there... Please review this awesome piece of code.

### CODE REVIEW

**Comment by skellerfenix** [comment]
@therisingpage this is cool, but it needs more comments!

File: risk_score.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+if __name__ == '__main__':
+    print 'This is the funky new Risk Score!!!!'
```
Comment (EmilyK) [inline risk_score.py]: Add more code Lynn!

**Comment by EmilyK** [comment]
@therisingpage I left a comment on your commit

**Comment by skellerfenix** [comment]
@therisingpage LGTM

File: risk_score.py (hunk 1 above)
Comment (therisingpage) [inline risk_score.py]: @EmilyK Grrrrr!!!

**Pull request merged**